interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -145,230 +143,227 @@
 <body>
 
  <div class="container">
-        <div class="row">
-            <div class="col-sm-12">
-                <h2>{xlt t="Vitals"}&nbsp;&nbsp;&nbsp;<a  href="../summary/demographics.php" onclick="top.restoreSession()" title="{xla t='Back to patient dashboard'}"><i id="advanced-tooltip" class="readonly fa fa-arrow-circle-o-left fa-2x small" aria-hidden="true"></i> </a></h2>
```

Comment:
Note that this line is a bit tricky since it is not used in the main vital form, but is used when clicking on Trend in vitals section in the patient summary page; which is needed for user to go back to the patient summary page:

![image](https://user-images.githubusercontent.com/278968/91369007-66f01000-e7bf-11ea-82bc-66b1fe44f491.png)

![image](https://user-images.githubusercontent.com/278968/91369048-85560b80-e7bf-11ea-86e1-bc5f0688e74e.png)

Reply:
Hi @bradymiller 

So, should I add the link to back to Dashboard page? like this?
![image](https://user-images.githubusercontent.com/22230889/91461184-44dbb800-e8bb-11ea-89b5-5e9fe1d52d86.png)

Reply:
hi @stu01509 ,

Note that the vitals script is run in 2 different contexts.

1. Context One. When run from the Patient Demographics screen when click `Trend` button in the vitals section. In this case the above line should show an icon, however I am betting the fontawesome upgrade we did awhile ago broke that icon.

![image](https://user-images.githubusercontent.com/278968/91524525-4b1a6600-e8b4-11ea-8275-2802412611ed.png)


2. Context Two. Is how the form is normally run from the encounter gui. And in this case the link/icon is hidden (since do not want user going back to the patient summary screen in that context).

-brady